src/api/app/helpers/webui/user_helper.rb

```diff
@@ -86,6 +86,17 @@ def activity_date_commits(projects)
     end
   end
 
+  def filter_message(params)
```

Comment:
This code is only considering the `Projects / Packages` filter, although we also need to consider the `Role` filter. See the screenshot below, the message is also wrong because it should be `This user is not involved in any project or package for the selected role`. Multiple roles can be selected, so to pluralize or not the word `role`, you can use [pluralize](https://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-pluralize).

![role-filter](https://user-images.githubusercontent.com/1102934/121525812-1158bd00-c9f9-11eb-904a-3888c7ba7941.png)